README.md

```diff
@@ -1,22 +1,15 @@
-# Substrate Cumulus Parachain Template
+# Base Parachain Template
 
-A new [Cumulus](https://github.com/paritytech/polkadot-sdk/tree/master/cumulus)-based Substrate node, ready for hacking ☁️..
+A new [Cumulus](https://github.com/paritytech/polkadot-sdk/tree/master/cumulus)-based Substrate node. 
 
-This project is originally a fork of the
-[Substrate Node Template](https://github.com/substrate-developer-hub/substrate-node-template)
-modified to include dependencies required for registering this node as a **parathread** or
-**parachain** to a **relay chain**.
+This project is a fork of the
+[Cumulus Parachain Template](https://github.com/paritytech/polkadot-sdk/tree/master/templates/parachain) from the [Polkadot SDK](https://github.com/paritytech/polkadot-sdk). 
 
-The stand-alone version of this template is hosted on the
-[Substrate Devhub Parachain Template](https://github.com/substrate-developer-hub/substrate-parachain-template/)
-for each release of Polkadot. It is generated directly to the upstream
-[Parachain Template in Cumulus](https://github.com/paritytech/polkadot-sdk/tree/master/cumulus/parachain-template)
-at each release branch using the
-[Substrate Template Generator](https://github.com/paritytech/substrate-template-generator/).
+It is used to generate parachains using [Pop CLI](https://learn.onpop.io/pop/v/pop-cli), an all-in-one tool for Polkadot development.
 
-👉 Learn more about parachains [here](https://wiki.polkadot.network/docs/learn-parachains), and
-parathreads [here](https://wiki.polkadot.network/docs/learn-parathreads).
+💥 Learn more about Pop CLI [here]( https://github.com/r0gue-io/pop-cli).
 
+👉 Learn more about parachains [here](https://wiki.polkadot.network/docs/learn-parachains).
 
-🧙 Learn about how to use this template and run your own parachain testnet for it in the
-[Devhub Cumulus Tutorial](https://docs.substrate.io/tutorials/v3/cumulus/start-relay/).
+🧙 Learn about how to use this template and run your own parachain testnet for it in Substrate's 
+[Build a parachain Tutorial](https://docs.substrate.io/tutorials/build-a-parachain/).
```

Comment:
Is not better to use ours? https://learn.onpop.io/pop/v/pop-cli/parachains/running-your-parachain

Reply:
Thanks, made some adjustments.